helix-core/src/syntax.rs

```diff
@@ -50,6 +54,9 @@ pub struct LanguageConfiguration {
     #[serde(default)]
     pub auto_format: bool,
 
+    #[serde(default = "default_completion_punctuation")]
+    pub completion_punctuation: Vec<String>,
```

Comment:
This part is already automatic and done via the LSP `trigger_characters`. rust-analyzer will register `.` for example.

Reply:
I had fix it by use Option<Vec<String>>